.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
```

Comment:
This line doesn't belong to this change, right? :)

Reply:
Well i thought it belongs here , as im adding folders and files that i suppose to ignore so  i figured that i have to ignore this folder.

Reply:
For simplicity, I'll accept this. But please re-read my e-mail about collaboration ;)